.github/workflows/cuda.yml

```diff
@@ -30,17 +30,17 @@ jobs:
         include:
           - method: wheel
             compiler: gcc
-            python_version: "3.11"
+            python_version: "3.10"
             cuda_version: "11.8.0"
             task: cuda
           - method: source
             compiler: gcc
-            python_version: "3.9"
+            python_version: "3.12"
```

Comment:
proposing swapping thise one and the `wheel` task, so we're always testing newest CUDA together with newest Python